_server_execute_js.js

```diff
@@ -7,26 +7,36 @@ module.exports = function() {
     */
     hybridVersion: function() {
       const ipcRenderer = require('electron').ipcRenderer;
+      const remote = require('electron').remote;
+      let pc = remote.getGlobal('pageCache');
+      let cachedValue = pc.get('_true_' + document.documentElement.outerHTML);
 
-      if (document.querySelectorAll('link[rel="import"]').length > 0) {
-        let html = document.cloneNode(true);
-        html.querySelector('body').removeAttribute('unresolved');
+      if(cachedValue == -1) {
+        if (document.querySelectorAll('link[rel="import"]').length > 0) {
+          let html = document.cloneNode(true);
+          html.querySelector('body').removeAttribute('unresolved');
 
-        let shadowPolymerScript = document.createElement('script');
-        shadowPolymerScript.innerText =
-            'window.Polymer = { dom: "shadow"}';
-        let imports = html.querySelectorAll('link[rel="import"]');
-        html.querySelector('head').insertBefore(shadowPolymerScript,
-          imports[0]);
+          let shadowPolymerScript = document.createElement('script');
```

Comment:
`const shadowPolymerScript = ` ...
I realize this is just newly indented, but pointing out const anyway :)